modules/ROOT/pages/errors/gql-errors/22NC4.adoc

```diff
@@ -5,11 +5,11 @@ error: data exception - a label cannot both identifying and implied. The label(s
 
 == Explanation
 When defining a graph type, labels and relationships must be assigned clear roles.
-A label cannot be both "identifying" (used to uniquely identify nodes) and "implied" (automatically assigned based on other schema rules) at the same time.
+A label cannot be both "identifying" (to uniquely identify nodes) and "implied" (automatically assigned based on other schema rules) at the same time.
```

Comment:
```suggestion
A label cannot be both "identifying" (uniquely identify nodes) and "implied" (automatically assigned based on other schema rules) at the same time.
```
the `to` sounds odd to me when removing the `used`

Reply:
```suggestion
A label cannot be both "identifying" (uniquely identifies nodes) and "implied" (automatically assigned based on other schema rules) at the same time.
```